ee/ucp/interlock/config/updates.md

```diff
@@ -30,6 +30,6 @@ performs a rolling deploy to update all extensions.
 
 ```bash
 $> docker service update \
-    --image interlockpreview/interlock@sha256:d173014908eb09e9a70d8e5ed845469a61f7cbf4032c28fad0ed9af3fc04ef51 \
-    interlock
+    --image docker/ucp-interlock@sha256:d173014908eb09e9a70d8e5ed845469a61f7cbf4032c28fad0ed9af3fc04ef51 \
```

Comment:
This digest is wrong as it doesn't exist for `docker/ucp-interlock`.  The latest image for 3.1.6 is `docker/ucp-interlock@sha256:9ba5e9ffd451b732d0e442fe8e5dc482580a196a8c98218f742b0d4395504854`.